planner/core/common_plans.go

```diff
@@ -226,6 +226,9 @@ func (e *Execute) OptimizePreparedPlan(ctx context.Context, sctx sessionctx.Cont
 	}
 
 	if prepared.SchemaVersion != is.SchemaMetaVersion() {
+		// schema change cached plan must be invalidated since cached plan does
+		// NOT have a "cache key" with schema version like plan cache
```

Comment:
How about rewriting the comment to the following one:
```go
// In order to avoid some correctness issues, we have to clear the
// cached plan once the schema version is changed.
```